src/components/icons/alert-message-o.tsx

```diff
@@ -0,0 +1,22 @@
+// THIS FILE IS AUTOGENERATED, DO NOT MODIFY MANUALLY
+/* tslint:disable */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License. v. 2.0. If a copy of the MPL was not distributed with this file.
+* You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import GenerateComponentForGraphic from './template'
+
+// Direct access to SVG
+export const Graphic = (
+  <svg width='32' height='32' xmlns='http://www.w3.org/2000/svg' xmlnsXlink='http://www.w3.org/1999/xlink'>
+    <defs>
+      <path d='M15.29 17.71c-.18-.19-.29-.45-.29-.71 0-.13.03-.26.08-.38.05-.13.12-.23.21-.33.09-.09.2-.16.33-.21.24-.1.52-.11.76 0 .13.05.24.12.33.21.09.09.16.2.21.33.05.12.08.25.08.38 0 .26-.11.52-.29.71-.19.18-.45.29-.71.29-.13 0-.26-.03-.38-.08a.933.933 0 0 1-.33-.21zM15 14v-4a1 1 0 0 1 2 0v4a1 1 0 0 1-2 0zm11 6H13c-.019 0-.035.01-.053.011-.071.004-.136.022-.205.041-.059.016-.117.028-.171.054-.055.026-.101.065-.152.102-.056.041-.111.08-.158.133-.013.014-.03.02-.042.035L10 23.149V21a1 1 0 0 0-1-1H6V7h20v13zm0-15H6c-1.103 0-2 .897-2 2v13c0 1.103.897 2 2 2h2v4a1 1 0 0 0 1.781.625L13.48 22H26c1.103 0 2-.897 2-2V7c0-1.103-.897-2-2-2z'
+      id='a' />
```

Comment:
Please convert this to not use defs with an explicit ID (especially with #a which will almost certainly exist on the page already if there are 2 such graphics). This looks like it can be converted to a `<path` directly in the SVG, with the `fill` and `fillRule` from the `<use /` made inline to the `<path`.

Reply:
good find, updated